src/algorithms/all_neighbors.c

```diff
@@ -83,19 +80,22 @@ AllNeighborsCtx *AllNeighborsCtx_New
 	ASSERT(M   != NULL);
 	ASSERT(src != INVALID_ENTITY_ID);
 
+	// make sure matrix is not bitmap or full
+	GxB_set(M, GxB_SPARSITY_CONTROL, GxB_SPARSE);
```

Comment:
Please move this line to `GxB_MatrixTupleIter_reuse` similar to `GxB_MatrixTupleIter_new`

Reply:
I was thinking that because GxB_set has switch case in it, it's better not to call it on every call to reuse.
Anyways I've made the change.